### PR TITLE
Install more dependencies required by packages

### PR DESCRIPTION
**Pull request by dhodovsk**
- dos2unix installed for mingw-win-iconv, openoffice.org-dict-cs_CZ, pstoedit, pyparsing, python-sphinx sgpio
- perl-devel installed for perl-Archive-Zip, perl-Data-OptList, perl-DBI, perl-Digest-SHA, perl-MRO-Compat, perl-File-Temp
- rubygems installed for rubygem-kramdown rubygem-rspec rubygem-rspec-core rubygem-thread_order rubygem-rspec-support rubygem-rspec-mocks rubygem-coderay
- gtk-doc installed for libappindicator, libiptcdata, libucil, libunicap, mingw-pango, orc

- Fixes https://github.com/packit/dist-git-to-source-git/issues/75

### CODE REVIEW

**Comment by dhodovsk** [comment]
Related to: https://github.com/packit/dist-git-to-source-git/issues/56

File: Containerfile [hunk 1]
```diff
@@ -9,7 +9,11 @@ ENV package_manager ${package_manager:-yum -y}
 RUN $package_manager -y install gcc git krb5-devel python3-devel python3-pip rpm-build && $package_manager -y clean all
 RUN curl --output /usr/bin/get_sources.sh https://git.centos.org/centos-git-common/raw/master/f/get_sources.sh && chmod +x /usr/bin/get_sources.sh
 # Tools required by the %prep section of some packages
-RUN $package_manager -y install \
+# PowerTools repo installed for gtk-doc
+RUN $package_manager -y install dnf-plugins-core && \
+    $package_manager -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    $package_manager config-manager --set-enabled PowerTools && \
+    $package_manager -y install \
```
Comment (lachmanfrantisek) [inline Containerfile:16]: Can we merge at least this one with other packages?

```suggestion
    $package_manager config-manager --set-enabled PowerTools && \
    $package_manager -y install \
    https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

**Review by lachmanfrantisek — COMMENTED** [review]
Thanks!

File: Containerfile (hunk 1 above)
Comment (dhodovsk) [reply Containerfile:16]: The repository needs to be installed first, then enabled :)

Comment (dhodovsk) [reply Containerfile]: Turns out we don't even need to install it.

File: Containerfile [hunk 2]
```diff
@@ -18,6 +21,11 @@ RUN $package_manager -y install \
     gettext-devel \
     sscg \
     libtool \
+    dos2unix \
+    gtk-doc \
+    perl-devel \
+    rubygems \
+    && $package_manager --enablerepo=PowerTools --setopt=PowerTools.module_hotfixes=true install javapackages-local \
```
Comment (dhodovsk) [inline Containerfile:28]: In separate command due to the risky option.

**Review by lachmanfrantisek — APPROVED** [review]

**Pull request merged**